### ISSUE TITLE
Issue on page /intro-pvlib.html

### ISSUE THREAD

**Issue by cesardd** [martavp/pv-textbook#11]
We should add here and maybe at the Welcome page that pvlib is not installed by default, and an additional separated cell has to be added at the beginning to install it:
!pip install pvlib

I don't know if there are other packages to be installed

**Comment by cesardd** [comment]
Maybe we can add this as comments in the notebooks where needed

**Comment by martavp** [comment]
Yes, that is a good idea. I was planning to explain this in the intro to pvlib (which still is work in progress), but we can also add a note to the notebooks
